sssom/cliques.py

```diff
@@ -103,9 +99,7 @@ def get_src(src, cid):
 
 
 def summarize_cliques(doc: MappingSetDataFrame):
-    """
-    summary stats on a clique doc
-    """
+    """Summarize stats on a clique doc."""
```

Comment:
Prefer one-line docstrings if it all fits rather than breaking up a logical one-line docstring into 3